backend/wmg/data/query.py

```diff
@@ -153,7 +155,17 @@ def _query(
 
         tiledb_dims_query = tuple(tiledb_dims_query)
 
-        query_result_df = cube.query(cond=query_cond or None, use_arrow=True).df[tiledb_dims_query]
+        attributes = ["cell_type_ontology_term_id"]
+
+        if not isinstance(criteria, FmgQueryCriteria) and criteria.compare_dimension is not None:
```

Comment:
the `compare` param isn't in the fmg queries, but i'm not 100% sure if it's correct to query just the `cell_type_ontology_term_id` attribute or if we should just not be passing in this param and querying all attributes